architecture.md

```diff
@@ -230,8 +230,7 @@ Most users will not need to modify the configuration file.
 The file is well commented and provides a few "knobs" that can be used to modify
 the behavior of the runtime.
 
-The configuration file is also used to enable runtime debug output (see
-some url to documentation on how to enable debug).
```

Comment:
Nice - can't believe we left 'some url' in there!

Reply:
@grahamwhaley : `Call into the post-start hooks. Usually, this is a no-op since nothing is provided (this needs clarification)` [here](https://github.com/kata-containers/documentation/blob/master/architecture.md#start) also needs to be removed after providing clarification?

Reply:
Ah, indeed @nitkon - we should try to clarify those.
@sboeuf - are you able to add any clarification to these two places in the arch doc:

https://github.com/kata-containers/documentation/blame/master/architecture.md#L278
```
2. Call into the post-start hooks. Usually, this is a no-op since nothing is provided
(this needs clarification)
```

https://github.com/kata-containers/documentation/blame/master/architecture.md#L347
```
1. Ask the container status by checking information stored on disk. (clarification needed)
```

If we can provide clarification and wording, maybe @nitkon would be kind enough to add the fixes to this PR :-)